commons-test/src/main/java/org/infinispan/commons/test/CommonsTestBlockHoundIntegration.java

```diff
@@ -19,7 +19,10 @@ public void applyTo(BlockHound.Builder builder) {
             // TestNG may be started on main thread and load this before renaming to testng
             t.getName().startsWith("main") ||
             // The threads our tests run on directly
+            t.getName().startsWith("TestNG") ||
             t.getName().startsWith("testng") ||
```

Comment:
Oh, I didn't realize you also changed commons-test/src/main/java/org/infinispan/commons/test/TestResourceTracker.java. In which case I think this can be removed then.